lib/mayaUsd/render/MaterialXGenOgsXml/libraries/mx_lighting_maya_v2.glsl

```diff
@@ -14,14 +14,13 @@ vec3 mx_environment_irradiance(vec3 N)
 vec3 mx_environment_radiance(vec3 N, vec3 V, vec3 X, vec2 alpha, int distribution, FresnelData fd)
 {
     N = mx_forward_facing_normal(N, V);
-    vec3 L = reflect(-V, N);
+    vec3 L = fd.refraction ? mx_refraction_solid_sphere(-V, N, fd.ior.x) : -reflect(V, N);
     float NdotV = clamp(dot(N, V), M_FLOAT_EPS, 1.0);
     float avgAlpha = mx_average_alpha(alpha);
     vec3 F = mx_compute_fresnel(NdotV, fd);
     float G = mx_ggx_smith_G2(NdotV, NdotV, avgAlpha);
-    vec3 comp = mx_ggx_energy_compensation(NdotV, avgAlpha, F);
+    vec3 FG = fd.refraction ? vec3(1.0) - (F * G) : F * G;
     float phongExp = mayaRoughnessToPhongExp(sqrt(avgAlpha));
     vec3 Li = mayaGetSpecularEnvironment(N, V, phongExp);
-
-    return Li * F * G * comp;
+    return Li * FG;
```

Comment:
Took the latest [mx_environment_prefilter.glsl](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/pbrlib/genglsl/lib/mx_environment_prefilter.glsl) from MaterialX and adapted it for Maya lighting.